### ISSUE TITLE
Eagle Schematic does not include config button 

### ISSUE THREAD

**Issue by jrohs** [IDMIL/T-Stick#1]
There is no reference to where the configuration button should be connected in the documentation or on the schematic. (Latest Firmware says Pin 15) but this should be shown in the schematic diagram and/or referenced in documentation.

**Comment by edumeneses** [comment]
Definitely, also, we simplified the FSR connection, and I'm advocating to remove the piezo. 

We'll have to redo the schematic to reflect all these changes.

**Comment by edumeneses** [comment]
Still open as we need to redo the schematics

**Comment by aburt2** [comment]
Issue moved to new documentation repository: https://github.com/IDMIL/tstick-docs
